### ISSUE TITLE
Usage w/o Emotion

### ISSUE THREAD

**Issue by ioanlucut** [davidhu2000/react-spinners#208]
First, thanks for the lib!

However, it uses Emotion (https://github.com/davidhu2000/react-spinners/blob/master/package.json#L93) which means an extra bundle of JS for any app which does not use that framework by default.

I would love to use this without having the extra payload in our bundle.

Would this be an option?

**Comment by mnlbox** [comment]
@ioanlucut Yeah, I have same issue. I prefer a smaller bundle size. Your final bundle size now is a little huge and not fit good for production:
https://bundlephobia.com/result?p=react-spinners@0.9.0

@davidhu2000 any plan to fix this issue and decrease bundle size?

**Comment by davidhu2000** [comment]
@ioanlucut can you provide more details on your application setup? so I can create a version locally to test. If you'd like more details on why emotion was added. It was done for cleaner code and server side rendering https://github.com/davidhu2000/react-spinners/issues/5

**Comment by davidhu2000** [comment]
@mnlbox, how are you importing the loaders? You can import them individually, which should help reduce bundle size. 

```
import BarLoader from 'react-spinners/BarLoader';`
```

It's also possibly related to https://github.com/davidhu2000/react-spinners/issues/188

**Comment by ioanlucut** [comment]
@davidhu2000 currently we use SCSS, so the emotion is not used at all in the codebase.

Thinking out loud: maybe a solution is to use emotion but provide a dist version of the spinners with the compiled inline CSS while tree shaking everything else.

**Comment by sn4f** [comment]
+1 bump

**Comment by davidhu2000** [comment]
this seems to be the most viable solution https://stackoverflow.com/a/71102920/6496010, but it'll require a ton of rewrites.

**Comment by davidhu2000** [comment]
ok, published alpha version for `BarLoader` without emotion, give it a try. `v0.13.0-alpha.1`

one note: you'd need to add 

```jsx
<BarLoader css={{ display: 'block' }} />
```

to have it show up properly. will be fixed later

**Comment by davidhu2000** [comment]
https://bundlephobia.com/package/react-spinners@0.13.0-alpha.3
<img width="328" alt="Screen Shot 2022-05-16 at 8 07 01 AM" src="https://user-images.githubusercontent.com/15827041/168624470-faa3e2b9-3bc0-4ff6-a93f-b1696a4efa38.png">

https://bundlephobia.com/package/react-spinners@0.12.0
<img width="320" alt="Screen Shot 2022-05-16 at 8 07 08 AM" src="https://user-images.githubusercontent.com/15827041/168624474-fbeff4d5-4cbe-496b-88e9-ea6e2f1e863d.png">

wow, removing emotion make a huge difference in size

**Comment by davidhu2000** [comment]
https://bundlephobia.com/package/react-spinners@0.13.0-alpha.3
<img width="328" alt="Screen Shot 2022-05-16 at 8 07 01 AM" src="https://user-images.githubusercontent.com/15827041/168624470-faa3e2b9-3bc0-4ff6-a93f-b1696a4efa38.png">

https://bundlephobia.com/package/react-spinners@0.12.0
<img width="320" alt="Screen Shot 2022-05-16 at 8 07 08 AM" src="https://user-images.githubusercontent.com/15827041/168624474-fbeff4d5-4cbe-496b-88e9-ea6e2f1e863d.png">

wow, removing emotion make a huge difference in size

**Comment by davidhu2000** [comment]
ok, this is published as `0.13.0-alpha.5`.

**Comment by davidhu2000** [comment]
going to close this real for now, please let me know if there are issues. hopefully release 0.13 by next week
